provision-contest/ansible/roles/mysql_server/tasks/main.yml

```diff
@@ -67,3 +67,23 @@
   loop:
     - load-db
     - dump-db
+
+- name: create mysql user for for DOMjudge database administration
+  mysql_user:
+    name: domjudge_dba
+    host: '{{ SERVER_IP_PREFIX }}.%'
+    password: "{{ DBA_PASSWORD }}"
+    append_privs: true
+    priv: 'domjudge.*:ALL,GRANT/*.*:CREATE USER,RELOAD'
+    state: present
+  when: DBA_PASSWORD is defined
```

Comment:
We only need to do this for groups['domserver'][0] I think? And we can even loop over all the domservers and create the user for that to make sure we don't open it in /24 but only as /29 or something.

Reply:
Yes

Reply:
Done